mathics/main.py

```diff
@@ -1,8 +1,16 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
+"""
+Simpler Command-line interface to Mathics3.
+
+See also mathicsscript for a more sophisticated, and full
+featured CLI, which uses in more add-on Python packages and modules
+to assist in command-line behavior.
+"""
 
 import argparse
 import atexit
+import cProfile
```

Comment:
I just wonder, is this package available by default?

Reply:
It is part of the standard library.